experimental/puppeteer-firefox/lib/ExecutionContext.js

```diff
@@ -21,9 +21,34 @@ class ExecutionContext {
       });
       return createHandle(this, payload.result, payload.exceptionDetails);
     }
+    if (typeof pageFunction !== 'function')
+      throw new Error('The following is not a function: ' + pageFunction);
```

Comment:
Say what we tried to do please. "Cannot evaluate".

The error:
"The following is not a function: 56" looks pretty unhelpful.

Reply:
sure, reworded both here and in Puppeteer